GPT/gpt.py

```diff
@@ -113,6 +115,26 @@ def add_to_confirmation_gui(model_output: str):
         GPTState.text_to_confirm = model_output
         confirmation_gui.show()
 
+    def gpt_clear_context():
+        """Reset the stored context"""
+        clear_context()
+
+    def gpt_new_thread():
+        """Create a new thread"""
+        new_thread()
+
+    def gpt_optimize_context():
+        """Optimize the reused context to save tokens"""
+        optimize_context()
+
+    def gpt_optimize_thread():
+        """Optimize the thread to save tokens"""
+        optimize_thread()
+
+    def gpt_push_context(context: str):
+        """Add the selected text to the stored context"""
+        push_context(context)
```

Comment:
Might not be necessary to extract the context helper functions as a separate function if we aren't doing anything but call that function. But perhaps I am missing the background.

Reply:
I did that to isolate the global usage, mostly. It seemed cleaner to me.